src/main/java/com/pingcap/tikv/PDClient.java

```diff
@@ -301,26 +307,30 @@ private PDClient(TiSession session) {
     super(session);
   }
 
-  private void initCluster() {
-    GetMembersResponse resp = getMembers();
+  private void initCluster(List<HostAndPort> pdAddrs) {
+    GetMembersResponse resp = null;
+    for(HostAndPort u: pdAddrs) {
+      resp = getMembers(u);
+      if(resp != null) {
+        break;
+      }
+    }
     checkNotNull(resp, "Failed to init client for PD cluster.");
     long clusterId = resp.getHeader().getClusterId();
     header = RequestHeader.newBuilder().setClusterId(clusterId).build();
     tsoReq = TsoRequest.newBuilder().setHeader(header).build();
-    updateLeader(resp);
-    if (leaderWrapper == null) {
-      throw new TiClientInternalException("Error Updating leader.");
-    }
+    this.pdAddrs = pdAddrs;
+    createLeaderWrapper(resp.getLeader().getClientUrls(0));
     service = Executors.newSingleThreadScheduledExecutor();
-    service.scheduleAtFixedRate(() -> updateLeader(null), 1, 1, TimeUnit.MINUTES);
+    service.scheduleAtFixedRate(this::updateLeader, 1, 1, TimeUnit.MINUTES);
   }
 
   static PDClient createRaw(TiSession session) {
     PDClient client = null;
     try {
       client = new PDClient(session);
       client.setIsolationLevel(IsolationLevel.RC);
-      client.initCluster();
+      client.initCluster(session.getConf().getPdAddrs());
```

Comment:
little bit odd here since client already owns a session ref. Why need to pass in conf again.